Source/WebKit/UIProcess/Automation/SimulatedInputDispatcher.h

```diff
@@ -59,10 +59,11 @@ using VirtualKeyMap = HashMap<VirtualKey, VirtualKey, WTF::IntHash<VirtualKey>,
 #if ENABLE(WEBDRIVER_KEYBOARD_GRAPHEME_CLUSTERS)
 // A CharKey must only ever represent a single unicode codepoint or a single grapheme cluster.
 using CharKey = String;
+using CharKeySet = ListHashSet<String>;
 #else
-using CharKey = UChar32;
+using CharKey = char32_t;
+using CharKeySet = ListHashSet<uint32_t>;
```

Comment:
Can we not make a ListHashSet of char32_t?

Reply:
We can if we add the appropriate hash traits for `char32_t`; it seems like it sort of works on some platforms, but not on the GTK and WPE builds, at least. There is little benefit in adding all those traits, but I would be willing to do that if you think it can’t wait. I started getting worried about the patch getting more and more complex.

Reply:
I left this as-is for now.